app/lib/actions/katello/repository/destroy.rb

```diff
@@ -52,9 +53,10 @@ def handle_redhat_content(repository)
           end
         end
 
-        def delete_record(repository)
+        def delete_record(repository, options = {})
           repository.destroy!
           repository.root.destroy! if repository.root.repositories.empty?
+          ::Katello::DockerMetaTag.cleanup_tags if options[:docker_cleanup]
```

Comment:
It would add the DockerMetaTag cleanup method to flow of to all repo deletes..might get confusing when the query appears in logs when deleting other types of repos?

Reply:
True. I could go either way :)